apps/common/lib/lexical/document/store.ex

```diff
@@ -70,21 +76,21 @@ defmodule Lexical.Document.Store do
       end
     end
 
-    def get_and_update(%__MODULE__{} = store, uri, updater_fn) do
+    def get_and_update(uri, updater_fn) do
       with {:ok, document} <- fetch(uri),
            {:ok, updated_source} <- updater_fn.(document) do
         ets_put(uri, :sources, updated_source)
 
-        {:ok, updated_source, store}
+        {:ok, updated_source}
       else
         error ->
           normalize_error(error)
       end
     end
 
-    def update(%__MODULE__{} = store, uri, updater_fn) do
-      with {:ok, _, new_store} <- get_and_update(store, uri, updater_fn) do
-        {:ok, new_store}
+    def update(uri, updater_fn) do
+      with {:ok, _} <- get_and_update(uri, updater_fn) do
+        :ok
```

Comment:
So this means after this pr, we won't update the store genserver state when call `get_and_update` or `update`, right?

Reply:
We weren't updating the genserver state before.